<?xml version="1.0"?>
<launch>
	<param name="service_name" value="server_led_com"/>
	<param name="path_device" value="/dev/ttyUSB_LEDS"/>
	
	<param name="topic_direction" value="/rb1_base/robotnik_base_control/cmd_vel"/>
	
	<param name="max_value_lineal_x" value="1"/>
	<param name="max_value_lineal_y" value="1"/>
	<param name="max_value_angular_z" value="1"/>
	<param name="porc_min_linear_x" value="0.03"/>
	<param name="porc_min_linear_y" value="0.03"/>
	<param name="porc_min_angular_z" value="0.05"/>
	<param name="leds" value="161"/>
	
	<param name="led_start_front" value="74"/>
	<param name="led_end_front" value="97"/>
	
	<param name="led_start_right" value="108"/>
	<param name="led_end_right" value="149"/>
	
	<param name="led_start_back" value="152"/>
	<param name="led_end_back" value="11"/>
	
	<param name="led_start_left" value="22"/>
	<param name="led_end_left" value="63"/>
	
	<param name="led_start_front_right" value="98"/>
	<param name="led_end_front_right" value="107"/>
	
	<param name="led_start_front_left" value="64"/>
	<param name="led_end_front_left" value="73"/>
	
	<param name="led_start_back_left" value="12"/>
	<param name="led_end_back_left" value="21"/>
	
	<param name="led_start_back_right" value="150"/>
	<param name="led_end_back_right" value="159"/>
	
	
	<node pkg="robotnik_arduino_led" type="service_leds_com" name="service_leds_com" output="screen"/>
	<node pkg="robotnik_arduino_led" type="node_direction_leds" name="node_direction_leds" output="screen"/>
	
</launch>
